docs/index.md

```diff
@@ -43,7 +43,8 @@ title: Trustroots Team Guide
 - [Angular Directives](Angular-Directives.md) (deprecated)
 - [App and server Monitoring](Monitoring.md)
 - [Database](Database.md)
-- [i18n](i18n.md)
+- [Internationalization (i18n)](i18n.md)
+- [Localization (l10n)](l10n.md)
```

Comment:
The difference between the two might be confusing to some; how about adding "— for translators" and "— for developers" after each title?

Reply:
:+1: Done in 45c0144083fe7ff3e4e46f1f991b95c7a8f29875